lenra-io/lenra_cli
# feat: Manage devtool log level

## About this PR

Closes #359 

Allow defining the devtool log level in the dev section of the `lenra.yml` file:

```yaml
dev:
  devtool:
    logLevel: debug
```

## Technical highlight/advice

If not defined will use the default value: `info` (defined if the devtool itself).

## How to test my changes
- Build the CLI
- add the upside YAML lines to the `lenra.yml` of a project
- run the net lines and you should see some debug logs:
```bash
lenra build
lenra start
lenra logs -f devtool
```

## Checklist

### I included unit tests that cover my changes

### I added/updated the documentation about my changes
